### PR TITLE
Iter2

### PR DESCRIPTION
**Pull request by angusev**

### CODE REVIEW

File: code/model.py [hunk 1]
```diff
@@ -25,7 +24,7 @@ def __init__(self):
         super(PrintLayer, self).__init__()
 
     def forward(self, x):
-        #         print(x.shape)
+        # print(x.shape)
```
Comment (mrartemev) [inline code/model.py]: Давай все таки удалим комменты

File: code/model.py [hunk 2]
```diff
@@ -57,6 +56,7 @@ def block(
             return layers
 
         self.main = nn.Sequential(
+            PrintLayer(),
```
Comment (mrartemev) [inline code/model.py]: Ты можешь использовать torchsummary чтобы не делать PrintLayer

File: code/model.py [hunk 3]
```diff
@@ -69,7 +69,8 @@ def block(
             PrintLayer(),
             *block(ngf * 2, ngf),
             PrintLayer(),
-            torch.nn.ConvTranspose2d(ngf, 3, 4, 2, 1, bias=False),
+            nn.Upsample(scale_factor=2, mode='bilinear'),
+            torch.nn.Conv2d(ngf, 3, 4, padding=2, bias=False),
```
Comment (mrartemev) [inline code/model.py]: Плохая идея - оставлять одну свертку после апсемплинга. У нее не сильно мощности хватит чтобы сделать тебе красивое финальное изображение

**Comment by MaximArtemev** [comment]
Еще, давай попробуем использовать свертки с kernel = 3 и padding = 1
